src/couch_mrview/src/couch_mrview_util.erl

```diff
@@ -533,6 +533,7 @@ apply_limit(ViewPartitioned, Args) ->
 
 validate_all_docs_args(Db, Args0) ->
     Args = validate_args(Args0),
+    check_range_all_docs(Args),
```

Comment:
why not put this inside validate_args?

Reply:
Tried that before but it fails in mango tests related to `_find` endpoint.
https://ci-couchdb.apache.org/blue/organizations/jenkins/jenkins-cm1%2FPullRequests/detail/PR-4518/9/pipeline/
```
ERROR: test_gte_respsects_unicode_collation (03-operator-test.OperatorJSONTests)
----------------------------------------------------------------------
Traceback (most recent call last):
 File "/home/jenkins/workspace/jenkins-cm1_PullRequests_PR-4518/25.3/build/apache-couchdb-3.3.2-04ead65/src/mango/test/03-operator-test.py", line 187, in test_gte_respsects_unicode_collation
   docs = self.db.find({"ordered": {"$gte": "a"}})
 File "/home/jenkins/workspace/jenkins-cm1_PullRequests_PR-4518/25.3/build/apache-couchdb-3.3.2-04ead65/src/mango/test/mango.py", line 280, in find
   r.raise_for_status()
 File "/home/jenkins/workspace/jenkins-cm1_PullRequests_PR-4518/25.3/build/apache-couchdb-3.3.2-04ead65/src/mango/.venv/lib/python3.9/site-packages/requests/models.py", line 960, in raise_for_status
   raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: http://127.0.0.1:15984/mango_test_cad95f52e95341e097ce2c1d861c1457/_find
======================================================================
ERROR: test_old_selector_with_no_selector_still_supported (16-index-selectors-test.IndexSelectorJson)
... ...
```